Add tremulous port

lyapova opened the pull request.

grayed commented:
I'll commit the port, and here are some comments that should be addressed. Please note that even in the case port will be polished, it's going to miss 5.8 release since we're too close to lock.

I can't build the port myself for now (no broadband Internet connection available temporary). So the comments below are not exhaustive.

1. PERMIT_PACKAGE_FTP and PERMIT_DISTFILES_FTP lines could be zapped. BTW, those should be catched by portcheck(1) - did you run it, as porting FAQ suggests?

2. We do not use RMD160, MD5 and SHA1 for a long time. Did you started your port based on one from another OS?

3. Instead of INSTALL_SCRIPT+perl dance you can use SUBST_VARS and SUBST_PROGRAM instead.

4. Please use built-in "getopts" instead of external "getopt". The latter is error-prone and its usage is, generally speaking, discouraged.

5. The patch-scr_cmdlib fixes a real problem, right? Did the fix went upstream? Could you, please, add a comment?

6. Are tremulous-bin and tremulous-dedicated binary executables? If yes, please run "make update-plist" after building port, those items should be prepended with "@bin" marker after that. If not, please report so.

lyapova commented:
1. Fixed. Yea.
2. Fixed. It based on old OpenBSD port.
3. Fixed.
4. Fixed. Add new Wrapper's.
5. Yea. Fix compile with -Werror. Fixed in upstream.
6. Yes executables. Fixed.
